pkgs/applications/audio/quadrafuzz/default.nix

```diff
@@ -0,0 +1,41 @@
+{ stdenv, fetchFromGitHub, boost, cairo, lv2, pkgconfig }:
+
+stdenv.mkDerivation rec {
+  pname = "quadrafuzz";
+  version = "unstable-17-10-2019";
+
+  src = fetchFromGitHub {
+    owner = "jpcima";
+    repo = pname;
+    rev = "a7cc7cba0bef09da9e6443e8967366650a372c37";
```

Comment:
Any reason for not using the tagged `0.1.0` release? I know this commit is newer than the tag, but I want to know if this is worth it.

Reply:
@jpcima What's your take on this, latest commit or release?

Reply:
I've released `0.1.1` just now, you can use this instead

Reply:
Thanks, will do! :+1: